doc/release/release_0.16.rst

```diff
@@ -0,0 +1,389 @@
+Announcement: scikit-image 0.16.0
+=================================
+
+We're happy to announce the release of scikit-image v0.16.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+Starting from this release, scikit-image will follow the spirit of the recently
```

Comment:
What does it mean to follow the spirit? Why not just follow it? :)

Reply:
I noticed this too but then thought, hey, maybe we won't follow it to the letter, because reasons. But I agree that the wording is better without spirit, removing it.